solidity-v1/dashboard/src/css/forms.less

```diff
@@ -158,4 +158,8 @@ input {
     font-size: 0.875rem;
     line-height: 1.25rem;
   }
+
+  &__text {
+    width: 100%;
+  }
```

Comment:
Please see https://github.com/keep-network/keep-core/pull/2668/#discussion_r750252502

Reply:
I think you forgot to add commit hash here 😅

Reply:
lol I meant this comment https://github.com/keep-network/keep-core/pull/2668#discussion_r750252502

Reply:
Fixed in f927f0cb007131499a7581aac5eb6856e3334a61